src/insights_client/tests/test_client.py

```diff
@@ -24,13 +24,12 @@ def test_keyboard_interrupt(os_uid, client):
 @patch('os.getuid', return_value = 0)
 @patch('insights.client.phase.v1.get_phases')
 @patch('insights.client.InsightsClient')
-@patch('insights_client.sorted_eggs', return_value = "/var/lib/insights/newest.egg")
 @patch('insights_client.subprocess.Popen')
 @patch('insights_client.enumerate', return_value = [("1", "egg")])
 @patch('insights_client.os.path.isfile', return_value = True)
-def test_phase_error_100(isfile, enumerate, mock_subprocess, sorted_eggs, client, p, os_uid):
+def test_phase_error_100(isfile, enumerate, mock_subprocess, client, p, os_uid):
     with pytest.raises(SystemExit) as sys_exit:
         mock_subprocess.return_value.returncode= 100
         mock_subprocess.return_value.communicate.return_value = ('output', 'error')
-        insights_client.run_phase(p, client, sorted_eggs)
+        insights_client.run_phase(p, client, validated_eggs=[""])
```

Comment:
most likely `validated_eggs` can be an empty list; `run_phase()` picks `ENV_EGG` anyway

Reply:
#180 obsoletes this change